### ISSUE TITLE
why is eslint-plugin-react necessary?

### ISSUE THREAD

**Issue by tugorez** [cancunDevs/cancunDevs-back#4]
https://github.com/cancunDevs/cancunDevs-back/blob/develop/package.json#L30

**Comment by yoniihdc** [comment]
I think it isn't necessary, I'm going to fix that in a refactor to `package.json` related to [this](https://github.com/cancunDevs/cancunDevs-back/issues/3) issue. Good question.

**Comment by PabloFer13** [comment]
I dont think so, most react plugins are for `JSX` but we are not going to use it on the API
